packages/context/src/binding.ts

```diff
@@ -20,6 +20,8 @@ import {
 
 const debug = debugModule('loopback:context:binding');
 
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Why are we disabling this rule instead of fixing the linting violations as we do in other files?